source/domain-migration.txt

```diff
@@ -0,0 +1,76 @@
+================
+Domain Migration
+================
+
+The App Services base URL is being updated from ``https://realm.mongodb.com/``
+to a new URL: ``https://services.cloud.mongodb.com``.
+
+This change applies to the App Services UI, Admin API requests, and client
+API requests through the Atlas Device SDKs.
+
+Relevant SDK Versions
+---------------------
+
+Starting in the following versions, the SDKs will use the new base URL by
+default:
+
+- C++ SDK: version TBD
+- Flutter SDK: version TBD
+- Java SDK: version TBD
+- Kotlin SDK: version TBD
+- .NET SDK: version TBD
+- Node.js SDK: version TBD
+- React Native SDK: version TBD
+- Swift SDK: version TBD
+
+Begin Migration Now
+-------------------
+
+The new URL (``https://services.cloud.mongodb.com``) is available now and you
+can begin using it by specifying a base URL in your app configuration. You should
+also update your domain access lists to allow the new URL.
+
+The old URL (``https://realm.mongodb.com/``) will continue to work, but will be
+deprecated in the future. We recommend migrating to the new URL as soon as
+possible.
+
+Client Applications
+~~~~~~~~~~~~~~~~~~~
+
+Begin using the new URL by specifying it as the base URL in your App 
+Configuration. Refer to the SDK docs for details about specifying a base URL:
+
+- :ref:`C++ SDK <cpp-access-the-app-client>`
+- :ref:`Flutter SDK <flutter-app-client-configuration>`
+- :ref:`Java SDK <java-app-client-configuration>`
+- :ref:`Kotlin SDK <kotlin-app-client-configuration>`
+- :ref:`.NET SDK <dotnet-app-client-configuration>`
+- :ref:`Node.js SDK <node-access-the-app-client>`
+- :ref:`React Native SDK <react-native-app-client-configuration>`
+- :ref:`Swift SDK <ios-app-client-configuration>`
+
+Domain Access Lists
+~~~~~~~~~~~~~~~~~~~
+
+If you use a domain access list, add the new domain to this list to avoid
+service disruptions:
+
+.. code-block:: text
+
+   *.services.cloud.mongodb.com
+
+Regions-Specific Base URLs
```

Comment:
The formatting in this section might be a little off (there seems to be extra tick marks in the staging page):
<img width="798" alt="image" src="https://github.com/mongodb/docs-app-services/assets/24275071/27de5d20-9b95-468d-8352-72e9ff6e712b">

Reply:
Ooh, yes. Thank you!